server/src/users/policy.rs

```diff
@@ -0,0 +1,79 @@
+use crate::users;
+use anyhow::Result;
+use std::path::{Path, PathBuf};
+use tokio::fs;
+
+const POLICY_NAME: &str = "policy.rego";
+const REFERENCE_DATA_NAME: &str = "reference_data.json";
+
+fn workdir(dir: &Path, id: Option<String>, tee: String) -> PathBuf {
+    users::path(dir, id).join(tee)
+}
+
+pub async fn get_policy(dir: &Path, id: Option<String>, tee: String) -> Result<Option<String>> {
```

Comment:
Idiomatic Rust for getters would be `policy()`, not `get_policy`, which makes even more sense if this becomes a `User` method.
Same for `reference_data()` instead of `get_reference_data`.